assets/js/customizer.js

```diff
@@ -7,108 +7,124 @@
 				addItemButton = container.querySelector(".add-featured-item");
 			const { __, _x, _n, _nx } = wp.i18n;
 			let featuredArea,
-				itemObjects = new Map(),
-				timer,
+				settings_timer,
 				search_timer,
 				timer_ms = 500;
 
 			class ListItem {
-				constructor(post) {
-					this.key = post.ID;
+				constructor(post, parent) {
+					this.id = post.id;
```

Comment:
To make this back-compat, i.e. work with theme settings from the current version of the plugin, we could replace this line with something like:

`if ( post.hasOwnProperty('id') ) {
						this.id = post.id;
					} else if ( post.hasOwnProperty('ID') ) {
						this.id = post.ID;
					} else {
						this.id = null;
					}`

Reply:
maybe this should check for original_post_id instead

Reply:
`if ( post.hasOwnProperty('original_post_id') ) {
						this.id = post.original_post_id;
						post.id = this.id;
					} else if ( post.hasOwnProperty('id') ) {
						this.id = post.id;
					} else {
						this.id = null;
					}`